tests/src/Kernel/CompositeFieldsTest.php

```diff
@@ -52,7 +52,8 @@ protected function setUp(): void {
    */
   public function testCompositeOption(): void {
     // Create a test content type.
-    $type = $this->entityTypeManager->getStorage('node_type')->create(['name' => 'Test content type', 'type' => 'test_ct']);
+    $type = $this->entityTypeManager->getStorage('node_type')
```

Comment:
Because we don't delete revisions, we delete the whole entity and that is asserted that the entity is added in a past revision, and it has been deleted even though the latest revision the ref field was empty.

Reply:
Might be we should cover test case that we faced in ewcms - old revisions aren't deleted when entity is deleted. It will be also useful since we have 'composite_revisions' configuration, so behaviour should be different with this configuration is enabled and disabled.

Reply:
Clarified in a call.

Reply:
Yes, clarified.